sql/2020/20_Caching/cache_wastedbytes_lighthouse.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+# Distribution of bytes wasted (absence of adequate caching) from Lighthouse
+SELECT 
+  _TABLE_SUFFIX AS client,
+  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-long-cache-ttl.details.summary.wastedBytes")as numeric)/1024/1024) AS mbyte_savings,
+  COUNT(*) num_pages,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages       

```

Comment:
Nits:

- uppercase keywords `AS NUMERIC` and spacing around operators
- `COUNT(0)` and explicit `AS`
- "total" field
```suggestion
  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-long-cache-ttl.details.summary.wastedBytes") AS NUMERIC) / 1024 / 1024) AS mbyte_savings,
  COUNT(0) AS num_pages,
  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages
```